### PR TITLE
Outline first 5 sections of PRD

### PR DESCRIPTION
**Pull request by jgeaso1266**
## Key Changes
- [ ] added bullets to the first 5 sections of the PRD

## Notes
- section 5 is only partially completed

### CODE REVIEW

File: template/1-overview.md [hunk 1]
```diff
@@ -3,8 +3,14 @@
 *Maximum: 2 pages*
 
 *What is this project about?*
+- Event Radar is an app that helps to _easily_ facilitate connections with others
+- it allows users to discover and register for upcoming events in their area, and easily connect with friends they meet
+- it also allows event hosts to easily create and manage events, and even scan tickets for event goers
```
Comment (nikhen-s) [inline template/1-overview.md:8]: Suggestion: (maybe we can emphasize the local community part?)
Event Radar is an app that simplifies making connections in your local community by enabling users to discover and join nearby events, and easily connect with new acquaintances. It also allows users to create, manage and scan tickets at their own events —all through a versatile system featuring user-unique QR codes that enhances the experience for both organizers and attendees.

File: template/1-overview.md [hunk 2]
```diff
@@ -3,8 +3,14 @@
...
+- all of this is done through one all-purpose, unique QR code
 
 *Describe succinctly and precisely its purpose.*
+- provides a one-stop shop to find events, connect with new friends, message people, and even host events
+- removes the barriers to creating connections after the event (by scanning QR codes to add friends, you no longer need to remember someone's name the next day. Simply go to the app and you'll find their information easily.)
```
Comment (nikhen-s) [inline template/1-overview.md:14]: suggestion: By facilitating the easy scanning of QR codes to add friends, Event Radar not only simplifies keeping track of new connections but also encourages making them through direct, physical interactions

File: template/1-overview.md [hunk 3]
```diff
@@ -3,8 +3,14 @@
...
 
-*Include a 3-sentence (max) elevator pitch.*
-
+*Include a 3-sentence (max) elevator pitch.*  
+Event Radar is a versatile app that simplifies event discovery, connection-making, and hosting. Find nearby events, register effortlessly, and connect instantly with new friends—all through one unique QR code. Whether you're attending or hosting, Event Radar is the ultimate tool for social networking and event coordination, streamlining connections before, during, and after any gathering.
```
Comment (nikhen-s) [inline template/1-overview.md:18]: This is great!

File: template/2-history.md [hunk 4]
```diff
@@ -3,8 +3,40 @@
...
+- 4 epics:
+    - Epic 1: Allowing Users to Discover and Join Events
+    - Epic 2: Allowing Users to Interact With Others
+    - Epic 3: Allow Event Hosts to Easily Create Events
+    - Epic 4: Allow Events Hosts to Communicate with Participants Easily
+
+- we were able to achieve most of the user flows in epics 1-3
+
+*Epic 1*
+- home screen that shows all events available on the database, with search bars and filter buttons to filter the events to the user's liking
```
Comment (nikhen-s) [inline template/2-history.md]: suggestion: home screen that shows all events available on the database, `prioritising those nearby` ..

File: template/2-history.md [hunk 5]
```diff
@@ -3,8 +3,40 @@
...
+- home screen also features all the upcoming events that a user has registered for
+- when clicked, each event goes to full screen event details page (where a user can register for the event if not already registered)
```
Comment (nikhen-s) [inline template/2-history.md:17]: nit: maybe switch the order:
`when clicked, each event goes to full screen event details page (where a user can register for the event if not already registered)`
then,
 `home screen also features all the upcoming events that a user has registered for in a separate tab`

File: template/2-history.md [hunk 6]
```diff
@@ -3,8 +3,40 @@
...
+- once a user clicks on a message item in the messages list, they will see the full private chat history they have with a friend
+- once a user clicks on a friend list item, they will see the friend's profile
+- every user has a profile that they can edit in the profile screen
+- on signup, every user is given a unique QR code that they can use to access events and connect with friends
+
+*Epic 3*
+- the hosting screen is the dashboard for all event hosts
+- it features a list of all their active events which, when clicked, leads to a screen to edit those details
+- it allows hosts to add events easily
+- it also includes a map view for users who want to see their events that way
```
Comment (nikhen-s) [inline template/2-history.md]: nit: it also offers a map view, allowing hosts to visualize the locations of their events

File: template/2-history.md [hunk 7]
```diff
@@ -3,8 +3,40 @@
...
+- hosts get access to a tool that allows them to easily scan the QR codes of event goers and see if the event goers have signed up or not
 
 *What did you learn?*
+- many members of our team were new to the scrum process, so it was an adjustment period, especially at the beginning, to get used to agile development
+- often at the beginning we were so eager to correct issues with the code that we took on gigantic PRs that touched too many files, we learned from this mistake by 1) limiting the scope of PRs or splitting PRs if needed, and 2) writing better tasks in the SCRUM board that made the scope of the task clear to the developer
+- with an idea as ambitious as ours and a limited time to complete everything, we knew we would not be able to finish everything in time, so we had to make a lot of hard decisions regarding prioritization of tasks
```
Comment (nikhen-s) [inline template/2-history.md:37]: nit add: We learned to break down tasks in a way that reduces dependencies between them, and allows working our group members to work in parallel.

File: template/2-history.md [hunk 8]
```diff
@@ -3,8 +3,40 @@
...
 
 *What is missing to bridge from PoC to MVP?*
-
+- there are still some user flows that we designed that are missing from the PoC
+- some important defining features regarding connecting with others still need to be implemented
```
Comment (nikhen-s) [inline template/2-history.md:42]: question: does this need to be specific?

File: template/4-value_proposition.md [hunk 9]
```diff
@@ -3,8 +3,16 @@
 *Describe the value proposition and argue that it is:*
 
 *1. Easy to communicate*
+- One-stop platform for event discovery, registration, and connection-making.
+- Simplifies networking by enabling instant friend connections through QR code scanning.
+- Streamlines event hosting with efficient ticket scanning and attendee management tools.
 
 *2. Defensible*
+- Unique QR code-based networking feature sets it apart from traditional event apps.
```
Comment (nikhen-s) [inline template/4-value_proposition.md]: suggestions: (ChatGPT generated lol)
Distinctive QR code system that promotes genuine, physical networking, setting it apart from standard digital-only social platforms.

File: template/4-value_proposition.md [hunk 10]
```diff
@@ -3,8 +3,16 @@
...
+- Offers comprehensive event management tools that cater to both attendees and event hosts.
+- Focuses on enhancing post-event connections, creating a sustainable user base.
 
 *3. Relevant*
-
+- Meets the growing demand for seamless event experiences in a digital age.
```
Comment (nikhen-s) [inline template/4-value_proposition.md]: suggestion:
Meets the increasing need for localized and physically engaging event experiences in a predominantly digital social landscape.

File: template/5-mvp.md [hunk 11]
```diff
@@ -3,18 +3,41 @@
...
 
 ## User Stories and Key Features
 
 *User stories about how various personas will use the product in context.*
-
-*Identify and prioritise the key features required.*
+- As an event goer, I want to view a list of nearby events, so that I can decide which event to go to next to socialise and meet new people.
+- As an event goer, I want to have a unique identification ticket that allows me access to events I register for, so that I can easily access events and participate seamlessly.
+- As an event goer, I want to scan the QR codes of people I meet so that I can keep track of who I met and catch up with them later.
+- As an event goer, I want to see a list of people whose QR codes I have scanned in each event, so that I can easily recall who I met.
```
Comment (nikhen-s) [inline template/5-mvp.md]: If this is a key feature, and currently we can see friends list, it's probably better to include the feature where you can see which event that you met the friend from in v1? - nor maybe rephrase to:
`As an event goer, I want to view a list of people whose QR codes I have scanned at events, so that I can easily recall who I met.`

File: template/5-mvp.md [hunk 12]
```diff
@@ -3,18 +3,41 @@
...
 
 *Who are the target personas for this product?*
+- **Event Attendees**: Students and young professionals who are interested in discovering and attending events in their area, ranging from social gatherings to professional networking events.
+- **Event Hosts/Organizers**: Students, young professionals, or organizations responsible for planning and managing events, seeking tools to streamline event registration, ticketing, and attendee engagement, as well as promot their event to a young adult audience.
 
 *Which is the key persona?*
+- **University Student**: A university student who is actively involved in campus life and interested in exploring social and academic events both on and off campus. This persona values opportunities for networking, discovering new interests, and engaging in community activities.
 
 *High-level scenarios to adopt, use and share the product.*
+- **Adopting the Product**: A university student hears about Event Radar through a campus event promotion or fellow classmates. Intrigued by the idea of easily discovering and attending events, they download the app to explore upcoming activities relevant to their academic and personal interests.
```
Comment (nikhen-s) [inline template/5-mvp.md]: suggestion:
**Adopting the Product**: A university student hears about Event Radar through a campus event promotion or fellow classmates. Intrigued by the idea of easily discovering and attending events `all through one centralized app`, they download it to explore upcoming activities relevant to their academic and personal interests.

File: template/2-history.md [hunk 13]
```diff
@@ -3,8 +3,40 @@
...
+- some more user flows have to be thought through and designed to make sure they are the best for the user, such as group chats, in event geo-location, and history of attended events
```
Comment (nikhen-s) [inline template/2-history.md:43]: question: are we thinking to include group chats? i think it's good but ik it will be complex.
alternatively, we can just include notifications from the events.

**Review by nikhen-s — CHANGES_REQUESTED** [review]
Thank you so much for working on this! Left my suggestions :) if you disagree don't hesitate to reply to the comment

File: template/1-overview.md (hunk 1 above)
Comment (jgeaso1266) [reply template/1-overview.md:8]: ok, these are just bullets (the full written out version will come later), but I will add a bullet for that emphasis

File: template/2-history.md (hunk 4 above)
Comment (jgeaso1266) [reply template/2-history.md]: I'm not even sure if we have this feature available. There is a way to filter events, but we don't track user location yet so the events that people see are just all the events in the database in no particular order...

File: template/2-history.md (hunk 5 above)
Comment (jgeaso1266) [reply template/2-history.md:17]: the third bullet relates to both the first and second bullets (you can also see event details from the "upcoming events" screen

File: template/2-history.md (hunk 8 above)
Comment (jgeaso1266) [reply template/2-history.md:42]: It's not clear from the prompt, we can always flesh it out later.

File: template/4-value_proposition.md (hunk 10 above)
Comment (jgeaso1266) [reply template/4-value_proposition.md]: Ok, not seeing too much of a difference in your suggestion compared to what I put, but I'll add the word "localized"

File: template/2-history.md (hunk 13 above)
Comment (jgeaso1266) [reply template/2-history.md:43]: For MVP I would say no. It is not a key feature (though private chats are).

**Review by nikhen-s — APPROVED** [review]
LGTM
